docs/termux.md

```diff
@@ -7,78 +7,66 @@
+5. Set up NVM for multi-user. After installing NVM it automatically adds the necessary commands for it to work, but it will only work if you are logged in as root; which we do not want (see step 9) so do the following things.
 
    - Copy the lines NVM asks you to run after running the install script.
    - Run `nano /root/.bashrc` and comment out those lines by adding a `#` at the start.
    - Run `nano /etc/profile` and paste those lines at the end and make sure to replace `$HOME` with `/root`
    - Now run `exit` and start Debain again.
 
-7. After following the instructions and setting up NVM you can now install the [required node version](https://coder.com/docs/code-server/latest/npm#nodejs-version) using `nvm install version_here`.
-8. You don't need to install yarn.
-9. Now you are ready to install code-server, but let's switch users first.
-   - There are many answers on Stack Exchange for why logging in as root is not recommended,\
-     but here's a [short article](https://www.howtogeek.com/124950/htg-explains-why-you-shouldnt-log-into-your-linux-system-as-root/) for you.
-   - Run the following `visudo` to give your user (created in step 4) sudo privileges.
-   - After executing the command, scroll to `User privilege specification` and add `username ALL=(ALL:ALL) ALL`.
-10. To switch users, run `su - username`. **DO NOT forget to add the `-` between `su` and the username or, else the `/etc/profile` file won't be executed,** you may instead follow step 6 but edit the `/etc/bash.bashrc` file instead of `/etc/profile` if you don't want to add a `-` between `su` and username.
-
-11. To install `code-server` run the following.
-
-To check the install process (Will not actually install code-server)
+6. After following the instructions and setting up NVM you can now install the [required node version](https://coder.com/docs/code-server/latest/npm#nodejs-version) using `nvm install version_here`.
+7. To install `code-server` run the following.
+   > To check the install process (Will not actually install code-server)
+   > If it all looks good, you can install code-server by running the second command
 
 ```bash
 curl -fsSL https://code-server.dev/install.sh | sh -s -- --dry-run
 ```
 
-If all looks good, you can run to install code-server.
-
 ```bash
 curl -fsSL https://code-server.dev/install.sh | sh
 ```
 
-12. Every time you run `./start-debian.sh` (or `proot-distro login debian` if you used proot-distro), you can switch to your user `su - username` and run code-server.
+8. You can now start code server by simply running `code-server`.
+
+> Consider using a new user instead of root, read [here](https://www.howtogeek.com/124950/htg-explains-why-you-shouldnt-log-into-your-linux-system-as-root/) why using root is not recommended.\
+> Learn how to add a user [here](https://gist.github.com/arHSM/62242c343efc2827861ddc38e485d7df).
```

Comment:
I'm not against using a gist here but if you accidentally delete it in the future, it breaks our docs 😛

What if we move this into it's own section under Extra? And then you can use the text directly instead of linking the gist.

Reply:
Ok!